modules/t/genbank.t

```diff
@@ -47,6 +52,11 @@ is($parser->get_source,'Danio rerio (zebrafish)',"Testing get_source");
 is($parser->get_taxon_id,'7955',"Testing get_taxon_id");
 cmp_ok($parser->is_circular, '==', 0, "Testing is_circular");
 ok($parser->get_length == length($parser->get_sequence), "Testing length of the sequence");
+is(scalar @{$parser->get_db_xref_list_for_type('GeneID')}, 2, "Testing get_db_xref_list_for_type GeneID");
```

Comment:
Can you test the values of the xrefs you've fetched? I get nervous around trivial counts, as it could be working by coincidence. Furthermore, I'm not fond of comparing numbers with the string eq implicit to is(). At the very least it's a needless type coercion.